web/analysis/templatetags/key_tags.py

```diff
@@ -8,6 +8,25 @@ def getkey(mapping, value):
     if isinstance(mapping, dict):
         return mapping.get(value, "")
 
+#Added: Added function to get type of variable
+@register.filter(name="gettype")
+def gettype(value):
+    return str(type(value))
+
+#Added: Added function to split list
+@register.filter(name="split")
+def split(value, key):
+    return value.split(key)
+
+#Added: Added function to search for keys in nested dictionary
+@register.filter(name="findkey")
```

Comment:
not even used in code

Reply:
Ops, I did not removed it, let me change it thanks

Reply:
np, is why i mentioned yesterday, changed files, in that way you can see your real changes and accidental changes